### PR TITLE
[zh]Update content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md

### PR DESCRIPTION
**Pull request by nameYULI**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 1]
```diff
@@ -124,56 +124,91 @@ kubeadm upgrade plan [version] [flags]
...
 
 <tr>
-<td colspan="2">--ignore-preflight-errors stringSlice</td>
+<td colspan="2">--ignore-preflight-errors strings</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">检查清单，其错误将显示为警告。 例如：“IsPrivilegedUser，Swap”。 值 “all” 忽略所有检查的错误。</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+检查清单，其错误将显示为警告。 例如：“IsPrivilegedUser，Swap”。 值 “all” 忽略所有检查的错误。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md:187]: 英文原文呢？

**Comment by tengqm** [comment]
请补上英文原文。

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 2]
```diff
@@ -124,63 +27,129 @@ kubeadm upgrade plan [version] [flags]
...
+<td colspan="2">--ignore-preflight-errors strings</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">检查清单，其错误将显示为警告。 例如：“IsPrivilegedUser，Swap”。 值 “all” 忽略所有检查的错误。</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+A list of checks whose errors will be shown as warnings. Example: 'IsPrivilegedUser,Swap'. Value 'all' ignores errors from all checks.
+-->
+<p>
+其错误将显示为警告的检查列表。 例如：'IsPrivilegedUser，Swap'。 值 'all' 忽略所有检查的错误。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md:108]:
```suggestion
其错误将显示为警告的检查列表。 例如：'IsPrivilegedUser,Swap'。 值 'all' 忽略所有检查错误。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 3]
```diff
@@ -124,63 +27,129 @@ kubeadm upgrade plan [version] [flags]
...
+</td>
 </tr>
 
 <tr>
-<td colspan="2">--kubeconfig string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/admin.conf"</td>
+<td colspan="2">
+<!--
+--kubeconfig string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/admin.conf"
+-->
+--kubeconfig string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: "/etc/kubernetes/admin.conf"
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md:118]:
```suggestion
--kubeconfig string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值："/etc/kubernetes/admin.conf"
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md [hunk 4]
```diff
@@ -124,63 +27,129 @@ kubeadm upgrade plan [version] [flags]
...
+</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">与集群通信时使用的 kubeconfig 文件。 如果标志为未设置，则可以在一组标准位置中搜索现有的 kubeconfig 文件。</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+The kubeconfig file to use when talking to the cluster. If the flag is not set, a set of standard locations can be searched for an existing kubeconfig file.
+-->
+<p>
+与集群通信时使用的 kubeconfig 文件。 如果标志为未设置，则可以在一组标准位置中搜索现有的 kubeconfig 文件。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_plan.md:127]:
```suggestion
与集群通信时使用的 kubeconfig 文件。如果标志为未设置，则可以在一组标准位置中搜索现有的 kubeconfig 文件。
```

**Comment by nameYULI** [comment]
> /approve

Is the /lgtm tag missing?

**Comment by tengqm** [comment]
@nameYULI Please be patient. The convention is to have more than one reviewer to review each PR. For large changes, I'm waiting for another pair of eyes to screen defects.

**Comment by nameYULI** [comment]
@tengqm  ok, thank you very much

**Comment by nameYULI** [comment]
@mengjiao-liu  Hi, can you help me review this pr? it has been submitted for a while

**Comment by mengjiao-liu** [comment]
/lgtm
Thanks for your contribution!

**Pull request merged**